crates/gen/src/windows.rs

```diff
@@ -1,11 +1,25 @@
 /// Returns the build's `.windows` directory in the root of the workspace as a `PathBuf`.
 pub fn build_windows_dir() -> std::path::PathBuf {
-    let mut path = workspace_dir();
+    build_windows_dir_at_workspace(&workspace_dir())
+}
+
+/// Returns the build's `.windows` directory in the passed workspace directory as a `PathBuf`.
+pub fn build_windows_dir_at_workspace(workspace_dir: &std::path::PathBuf) -> std::path::PathBuf {
+    let mut path = workspace_dir.clone();
     path.push(".windows");
     path
 }
 
-fn workspace_dir() -> std::path::PathBuf {
+/// Returns the build's target directory in the passed workspace directory as a `PathBuf`.
+pub fn build_target_dir_at_workspace(workspace_dir: &std::path::PathBuf) -> std::path::PathBuf {
```

Comment:
`build_target_dir_at_workspace` => `workspace_target_dir`

The current wording is pretty confusing.

Reply:
`workspace_target_dir` sounds good.

Reply:
done - renamed it in last update